frontend/src/features/modules/ModulePanel.tsx

```diff
@@ -0,0 +1,12 @@
+import { useParams } from 'react-router-dom'
+
+export const ModulePanel = () => {
+  const { moduleName } = useParams()
+
+  const body = (
+    <div className='flex-1 py-2 px-4'>
+      <p>Module: {moduleName}</p>
+    </div>
+  )
+  return body
```

Comment:
We can probably just return the body directly here instead of creating a new `const body`